editor/components/inserter/menu.js

```diff
@@ -103,16 +107,16 @@ export class InserterMenu extends Component {
 		} );
 	}
 
-	selectBlock( name ) {
+	selectBlock( name, attributes ) {
 		return () => {
-			this.props.onSelect( name );
+			this.props.onSelect( name, attributes );
```

Comment:
Thinking more about this "attributes" argument. Regular blocks already have an `attributes` property but instead of containing `attributes` values, it contains `attributes` definition. I assume this will be called with the `attributes` definitions for these regular blocks which is not what's expected I guess. 

It may not produce bugs at the moment but I think we should fix this ambiguity somehow.

Reply:
👌 fixed in 8ced4510795d9e428a2f6d1c2bbb4e2f744da23b.